ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/platform/akkastreams/dispatcher/DispatcherSpec.scala

```diff
@@ -146,12 +145,13 @@ class DispatcherSpec
       })
 
   private val rangeQuerySteppingMode = RangeSource[Index, Value](
-    (startInclusive, endExclusive) => Source(store.get().range(startInclusive, endExclusive))
+    (startExclusive, endInclusive) =>
+      Source(store.get().from(startExclusive).to(endInclusive).filter(_._1 != startExclusive))
```

Comment:
Wouldn't `.drop(1)` do the trick? (And the same below.)

Reply:
Only of `startExclusive` is actually the first element. Maybe `dropWhile(_._1 <= startExclusive)` to be sure.